src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -285,3 +286,14 @@
             c.command('list', 'list_by_server')
             c.command('delete', 'delete')
             c.generic_update_command('update', 'get', 'create_or_update')
+
+    server_connection_policies_operations = create_service_adapter(
+        'azure.mgmt.sql.operations.server_connection_policies_operations',
+        'ServerConnectionPoliciesOperations'
+    )
+
+    with ServiceGroup(__name__, get_sql_server_connection_policies_operations, server_connection_policies_operations,
+                      custom_path) as s:
+        with s.group('sql server conn-policy') as c:
+            c.command('show', 'get')
+            c.generic_update_command('update', 'get', 'create_or_update')
```

Comment:
Well, I guess that's answers that.  Ha.  Would the command benefit from convenience arguments of do you feel the generic update is sufficient for your purposes? Most update commands benefit from stongly typed convenience args.

Reply:
There is a convenience argument `--connection-type`:

```
(env) D:\git\azure-cli [connPolicy ≡]> az sql server conn-policy update -h

Command
    az sql server conn-policy update: Updates a server's secure connection policy.

Arguments
    --connection-type -t [Required]: The server connection type.  Allowed values: Default, Proxy,
                                     Redirect.
    --resource-group -g  [Required]: Name of resource group. You can configure the default group
                                     using `az configure --defaults group=<name>`.
    --server -s          [Required]: Name of the Azure SQL server.
```